### PR TITLE
feat: Addition of Stt Units

### PR DESCRIPTION
**Pull request by Perseus333**
This PR adds the unit Stt (Seconds to Type) to the choosable typing units. STT is a unit which is measured in seconds of how long does it take you to type a 100 word (500 character) text.  It's a unit that is intended to see the real time saving or progress that you are archieving with your speed improvements. For more information see [this desmos](https://www.desmos.com/calculator/objfzho0ss).

### CODE REVIEW

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6187283763

**Comment by fehmer** [comment]
i think you're missing a change in the `backend/src/api/schemas/config-schema.ts` to account for the new  unit string

**Comment by fehmer** [comment]
do we want to add this unit to the settings page as well? `frontend/static/html/pages/settings.html`

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6187477326

**Comment by Perseus333** [comment]
> i think you're missing a change in the `backend/src/api/schemas/config-schema.ts` to account for the new unit string

oh, could be, since I did changes on frontend

**Comment by Perseus333** [comment]
> do we want to add this unit to the settings page as well? `frontend/static/html/pages/settings.html`

yeah I think so

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6189394787

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6223417215

**Comment by Miodec** [comment]
Account page is broken, please investigate.

**Review by fehmer — CHANGES_REQUESTED** [review]

File: frontend/src/ts/utils/typing-speed-units.ts [hunk 1]
```diff
@@ -22,7 +22,11 @@ class Unit implements MonkeyTypes.TypingSpeedUnitSettings {
   }
 
   fromWpm(wpm: number): number {
-    return wpm * this.convertFactor;
```
[the remark this answers is not in the document]
Comment (Perseus333) [reply frontend/src/ts/utils/typing-speed-units.ts:25]: It may have been creaated by prettify

[the remark this answers is not in the document]
Comment (Miodec) [reply frontend/src/ts/utils/typing-speed-units.ts:25]: Prettify doesn't add code, it only modifies formatting.

**Pull request closed without merging**